src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -321,8 +321,8 @@ static void LogMessageToPython(uint8_t module, uint8_t category, const char * ms
         gettimeofday(&tv, NULL);
 
         // Get the module name
-        char moduleName[nlChipLoggingModuleNameLen + 1];
-        ::chip:: ::Logging::GetModuleName(moduleName, module);
+        char moduleName[chip::Logging::kMaxModuleNameLen + 1];
+        ::chip:: ::Logging::GetModuleName(moduleName, sizeof(moduleName), module);
```

Comment:
nit (and existing code): this `::chip:: ::Logging` looks so odd ...

Reply:
@andy31415 Happy to change, any proposal? I can search replace ;)

Reply:
No idea. I am actually surprised it compiles. Good as is I guess.

Reply:
I actually am not convinced we even try to compile - I added some garbage code and our scripts still worked. Will try to mess with defines.